meta-refkit/conf/distro/include/refkit-ci.inc

```diff
@@ -70,7 +70,7 @@ REFKIT_CI_SDK_TARGETS=""
 # as possible.
 REFKIT_CI_ESDK_TARGETS=""
 # Following targets would be executed with do_test_iot_export task
-REFKIT_CI_TEST_EXPORT_TARGETS="refkit-image-common refkit-image-computervision refkit-image-gateway"
+REFKIT_CI_TEST_EXPORT_TARGETS="refkit-image-common refkit-image-computervision refkit-image-gateway refkit-image-industrial"
```

Comment:
My preference is to make it explicit in ```REFKIT_CI_BUILD_TARGETS```.  The same for refkit-image-computervision and refkit-image-gateway now that the the comment lines (which could be removed) before  ```REFKIT_CI_BUILD_TARGETS``` is obsolete.

Additionally, I think adding refkit-image-industrial here is not needed until we're ready to test it.